bciers/apps/reporting/src/app/components/finalReview/reviewDataFactory/facilityActivitiesFactoryItem.ts

```diff
@@ -0,0 +1,50 @@
+import activityFactoryItem from "./activityFactoryItem";
+import nonAttributableEmissionsFactoryItem from "./nonAttributableEmissionsFactoryItem";
+import emissionsSummaryFactoryItem from "./emissionsSummaryFactoryItem";
+import productionDataFactoryItem from "./productionDataFactoryItem";
+import allocationOfEmissionsFactoryItem from "./allocationOfEmissionsFactoryItem";
+import { ReviewData } from "./factory";
+
+export default async function facilityActivitiesFactoryItem(
+  versionId: number,
+  facilities: {
+    facility_id: string;
+    facility__name: string;
+    is_selected: boolean;
+  }[],
+): Promise<ReviewData[]> {
+  const reviewData: ReviewData[] = [];
+
+  for (const facility of facilities) {
+    if (facility.is_selected) {
+      reviewData.push({
+        schema: {
+          type: "object",
+          title: `Report Information - ${facility.facility__name}`,
+          properties: {},
+        },
+        uiSchema: {},
+        data: {},
+        isCollapsible: true,
```

Comment:
I think `isCollapsible` is redundant with `items` existing - keeping both around means we have a few more edge case to handle that are bug prone